lib/workers/repository/update/branch/index.ts

```diff
@@ -181,11 +178,13 @@ export async function processBranch(
           if (dependencyDashboardCheck || config.rebaseRequested) {
             logger.debug('Manual rebase has been requested for PR');
           } else {
-            const newBody = branchPr.body?.replace(
-              rebasingRegex,
-              '**Rebasing**: Renovate will not automatically rebase this PR, because other commits have been found.'
-            );
-            if (newBody !== branchPr.body) {
+            const newBody = await getPrBody(branchConfig, {
+              appendExtra: branchPr.bodyStruct?.reviewableSection,
```

Comment:
@rarkins Reviewable section is being restored here

Reply:
However, it won't be taken into account inside further `hashBody()` call

Reply:
Was it being restored to the PR body *before* though?

Reply:
Can you please elaborate more on what scenario you mean?

Reply:
The main reason we need any reviewable awareness at all is because it pollutes the PR body. I just did a GitHub search and it seems it still works this way:

![image](https://user-images.githubusercontent.com/6311784/167097659-bf6cae8b-3347-46cf-b7e0-b18575aa373f.png)

Without reviewable awareness, we'd keep pushing our same PR body every single run, and reviewable would edit it again, etc - flip flopping.

To avoid this, we strip out the reviewable content before comparing PR bodies.

But if we need to update the PR body for some other reason - e.g. change of versions or some other PR body content change - then I think it's ok for us to wipe away the reviewable comment again. Therefore we don't need to save/restore the content if we're making PR body updates.

Reply:
Thanks for clarification, now it's just being ignored at hashing phase without any storage